### ISSUE TITLE
Set up nginx for the server OSRM instance

### ISSUE THREAD

**Issue by thcrock** [ifed3/PBR#8]
We have an OSRM instance in a screen session on the server, but this is tough to automatically restart when we make changes. Rather we should daemonize it so after an update we can call 'sudo service osrm restart' or something.

Also, we want to expose it to the world so we can run requests from our local computers.

As it happens, we can kill two birds with one stone by setting up nginx on the server. 

https://www.digitalocean.com/community/tutorials/how-to-set-up-an-osrm-server-on-ubuntu-14-04

A lot of the work from that tutorial is already done, but the latter part of it we should do.

**Comment by ifed3** [comment]
Let's set up nginx as a reverse proxy [alongside](http://www.sitepoint.com/configuring-nginx-ssl-node-js/) node.js. We can operate nginx primarily as a frontend server providing static styling and mapping assets. Then utilize node to serve routing, dynamic data and as a [daemon](http://howtonode.org/deploying-node-upstart-monit) handling OSRM recycles.

**Comment by ifed3** [comment]
can you have coassignees on an issue?

**Comment by thcrock** [comment]
Doesn't look like it, but you can create a new issue that you own which covers the node part. I'll get nginx up and running first, then you can add node?

**Comment by thcrock** [comment]
Done! Here's what I did:
1. sudo yum install nginx
2. Added OSRM config file (from the guide) to /etc/nginx/conf.d instead of sites-{available,enabled}, because that's how it works on Amazon Linux
3. Installed supervisor via pip, because that's how it works on Amazon Linux
4. echo_supervisord_conf > /etc/supervisor/conf.d/osrm.conf
5. Added the guide's OSRM config file to the end of that conf file
6. supervisord -c /etc/supervisor/conf.d/osrm.conf

Sadly when installed through PIP it's not a service, so this won't persist across server restarts...but, we can restart using 'supervisorctl reload', so that is great.
